<?xml version="1.0"?>
<launch>
	<arg name="local_ip" default="192.168.31.162" />
	<!-- 第1台无人机 -->
	<group ns="/sunray_rmtt/rmtt_1">

		<!-- 启动rmtt_control_node -->
		<node pkg="sunray_rmtt" type="rmtt_control_node" name="rmtt_control_node_1" output="screen">
			<param name="uav_id" value="1" />
			<param name="uav_ip" value="192.168.31.240" />
			<rosparam command="load" file="$(find sunray_rmtt)/launch/rmtt_params.yaml" />
		</node>
	</group>

	<!-- 第2台无人机 -->
	<group ns="/sunray_rmtt/rmtt_2">

		<!-- 启动rmtt_control_node -->
		<node pkg="sunray_rmtt" type="rmtt_control_node" name="rmtt_control_node_2" output="screen">
			<param name="uav_id" value="2" />
			<param name="uav_ip" value="192.168.31.80" />
			<rosparam command="load" file="$(find sunray_rmtt)/launch/rmtt_params.yaml" />
		</node>
	</group>

	<!-- 第3台无人机 -->
	<group ns="/sunray_rmtt/rmtt_3">

		<!-- 启动rmtt_control_node -->
		<node pkg="sunray_rmtt" type="rmtt_control_node" name="rmtt_control_node_3" output="screen">
			<param name="uav_id" value="3" />
			<param name="uav_ip" value="192.168.31.14" />
			<rosparam command="load" file="$(find sunray_rmtt)/launch/rmtt_params.yaml" />
		</node>
	</group>

</launch>
